proto/forwarding/forwarding_port.proto

```diff
@@ -222,3 +223,35 @@ message PortStateRequest {
 message PortStateReply {
   PortInfo status = 1;
 }
+
+message HostPortControlRequest {
+  oneof msg {
+    ContextId context_id = 1;
+    google.rpc.Status status = 2;
+  }
+}
+
+message HostPortControlMessage {
+  PortDesc port = 1;
+  uint64 port_id = 2;
+  PortId dataplane_port = 3; // ID of the dataplane port related to this one.
+  bool create = 4;
```

Comment:
How is create used?

Reply:
Set to true if an hostif should be created, false for deletion.